cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,34 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+func tryWindowsAppDataMigration() {
+	if runtime.GOOS != "windows" {
+		return
+	}
+
+	localAppData := filepath.Join(os.Getenv("LocalAppData"), "Tailscale")
+	_, err := os.Stat(localAppData)
+	if os.IsNotExist(err) {
+		return
+	}
+	if err != nil {
+		log.Printf("tryWindowsAppDataMigration failed; no AppDataLocal: %v", err)
```

Comment:
"no" isn't quite right... if it were "no" you would've returned on 108. it's likely permissions here.

Reply:
Changed to "stat failed"

Reply:
"stat failed" is redundant with its own stringification of the error from stat:

```
bradfitz@tsdev:~$ cat stat.go 
package main

import (
        "fmt"
        "os"
)

func main() {
        _, err := os.Stat("/root/.ssh")
        fmt.Printf("foo: %v\n", err)
}
bradfitz@tsdev:~$ go run stat.go 
foo: stat /root/.ssh: permission denied
```